py/examples/table_events_select.py

```diff
@@ -0,0 +1,36 @@
+# Table / Events / Select
+# Register the `select` #event to emit Wave event on each #table row selection.
+# #table #events #select
+# ---
+from h2o_wave import main, app, Q, ui
+
+@app('/demo')
+async def serve(q: Q):
+    q.page['meta'] = ui.meta_card(
+    box='',
+    layouts=[
+        ui.layout(breakpoint='xs', zones=[
+            ui.zone('main', direction=ui.ZoneDirection.ROW, zones=[
+                ui.zone('table', size='50%'),
+                ui.zone('selected', size='50%')
+            ])
+        ])
+    ])
```

Comment:
Let's keep the examples as simple as possible by using the grid layout as the rest of the examples do.